upgrade/UPGRADE-v12.1.0-dev.md

```diff
@@ -0,0 +1,6 @@
+# [Upgrade from v12.0.0 to v12.1.0-dev](https://github.com/shopsys/shopsys/compare/v12.0.0...12.0)
```

Comment:
Shouldn't be the branch named 12.1?

Reply:
Hmm, interesting idea, I have created branch and changed this.